processors/processSkyBlock.js

```diff
@@ -25,6 +25,8 @@ async function checkItems(members = {}) {
         if (![null, undefined].includes(item.attributes.modifier)
           || id.startsWith('MAP:')
           || item.name === '§fnull'
+          || item.name.includes('✪')
+          || item.name.includes('Thick ')
```

Comment:
this should be changed to use `startsWith` just in case

Reply:
will do!